### PR TITLE
Create diagrams-and-formulae.md

### LINKED ISSUE #747: User guide: move features like TeX and diagram support to a separate section
Latex, Katex, Mermaid, PlanUML and MindMap are currently documented under [**Look and Feel**](https://www.docsy.dev/docs/adding-content/lookandfeel/). There is much more to these extensive features than just **look and feel** customization.

So I'd propose that they be moved to another section, possibly a top-level **Features**. It might even make sense to have **Latex support** and **Support for diagrams** be top-level sections. WDYT @LisaFC and @emckean?

### PR DESCRIPTION
**Pull request by LisaFC**
Split diagrams and formulae content from Look and Feel page to their own page. Fixes #747

### CODE REVIEW

**Comment by LisaFC** [comment]
@chalin, PTAL!

**Review by chalin — APPROVED** [review]
Super!

File: userguide/content/en/docs/Adding content/diagrams-and-formulae.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
+---
+title: "Diagrams and Formulae"
+linkTitle: "Diagrams and Formulae"
+weight: 11
+description: >
+  Add generated diagrams and scientific formulae to your site.
```
Comment (chalin) [inline userguide/content/en/docs/Adding content/diagrams-and-formulae.md:6]: Nit. Consider using the following simplified but equivalent front matter:

```yaml
title: Diagrams and Formulae
weight: 11
description: Add generated diagrams and scientific formulae to your site.
```

Comment (LisaFC) [reply userguide/content/en/docs/Adding content/diagrams-and-formulae.md]: Done.

**Pull request merged**